### DISCUSSION TITLE
Are "Discussions" a good thing?

### DISCUSSION THREAD

**Discussion by DanHickstein** [PyAbel/PyAbel#344] [General]
In the process of disabling the Wiki (I was surprise this was still around!), I saw that we could enable "Discussions". According to GitHub "Discussions is the space for your community to have conversations, ask questions and post answers without opening issues."

I can see how some people might want to ask questions without opening an Issue, since even the word "Issue" makes it seems like you reporting a bug or problem with the software. But, Issues has so far worked pretty well for our discussions.

Anyway, I activated this feature, so if you want to use it, please do!

**Comment by MikhailRyazanov** [comment]
GitHub has advertised this to me a month ago or so, and I thought that this might useful but haven't done anything. I'm glad that you did. :–)

I've converted #338 from an issue to a discussion for testing purposes... If it seems reasonable, we can also convert #341 and #236 (unless they can be closed already). Is “Q&A” the appropriate category for such questions or we should rename it/add another?

**Reply by DanHickstein** [reply]
Yeah, I suppose it would make sense for all of the conversations that don't actually relate to changes in the PyAbel code-base to become discussions rather than issues. And it seems like they can be migrated back and forth between discussions and Issues. I guess I see a few potential problems with using Discussions:
1. We need to keep track of conversations in two separate locations.
2. Discussions seems to use the "Reddit" style of commenting with replies to specific comments, and the comments can be sorted in any order. This is just different than the strictly chronological way that Issues work.
3. Will GitHub preserve discussions as reliably as they do Issues? We have some great discussions on here, and hopefully Discussions won't be discontinued at some point in time and all of the info lost. 

Anyway, I don't think that those are huge concerns, so let's keep trying this Discussions thing :)
